lib/image-transport.ts

```diff
@@ -0,0 +1,13 @@
+import { ImageTransport } from "./types";
+
+export function getImageTransport(targetImage: string): ImageTransport {
+  if (targetImage.startsWith("docker-archive:")) {
+    return ImageTransport.DockerArchive;
+  }
+
+  return ImageTransport.ContainerRegistry;
+}
+
+export function getDockerArchivePath(targetImage: string): string {
```

Comment:
does it have to be a Docker archive? or any image archive?

Reply:
I think so, the format of this tar is specifically named "docker archive" and "targetImage" is going to contain the prefix `docker-archive:`